lib/spell-check-view.coffee

```diff
@@ -107,3 +111,76 @@ class SpellCheckView
     instance = @getInstance()
     misspelling = @editor.getTextInBufferRange marker.getBufferRange()
     instance.suggest args, misspelling
+
+  initializeContextMenuListeners: =>
+    atom.views.getView(@editor).addEventListener 'mousedown', @addContextMenuEntries
```

Comment:
Ah right. I assumed that context menus were only opened with right-mouse clicks so thought a `mousedown` listener would work. I'll look into a more reliable way of detecting when context menus open/close.